docs/eventing/configuration/sugar-configuration.md

```diff
@@ -0,0 +1,47 @@
+# Configure Sugar Controller
```

Comment:
You need to add this to the navigation menu [nav.yaml](https://github.com/knative/docs/blob/main/config/nav.yml#L208)